src/Core/Framework/Plugin/Command/PluginListCommand.php

```diff
@@ -18,6 +18,31 @@ class PluginListCommand extends Command
 {
     protected static $defaultName = 'plugin:list';
 
+    /**
+     * @var int
+     */
+    protected $installed;
```

Comment:
the command should not have a state. just leave this numbers as local variables and pass them to the methods

Reply:
Why is that better?

Reply:
ok, for a console command this might not be a big thing...